libs/blocks/section-metadata/section-metadata.css

```diff
@@ -226,6 +226,37 @@ main > .section[class*='-up'] > .content {
   max-width: unset;
 }
 
+/* show-more-button */
+.section .show-more-button {
+  display: flex;
+  justify-content: center;
+  padding: 6px 0;
+  gap: 10px;
+  border-radius: 8px;
+}
+
+.section .show-more-button button {
+  background-color: var(--color-gray-100);
+  border: none;
+  color: #070707;
+  text-decoration: none;
+  cursor: pointer;
+  font-size: 18px;
```

Comment:
Why not `--type-body-m-size`. When an eventual redesign comes, it will be easier to make global updates, rather than having to edit each file.